### PR TITLE
CI/CD - Leveraging new reusable workflow in Standard repo deployment

### PR DESCRIPTION
**Pull request by Gordonby**
## PR Summary

This repo regularly tests 3 main configurations on PR/Schedule/Manually.

In order to test consuming the new reusable workflow as part of one of these Action Workflows, several changes had to be made.

## PR Checklist

- [ ] PR has a meaningful title
- [ ] Summarized changes
- [ ] This PR is ready to merge and is not **Work in Progress**
- [ ] Link to a filed issue
- [ ] Screenshot of UI changes (if PR includes UI changes)

### CODE REVIEW

File: .github/workflows/AKSC_Deploy.yml [hunk 1]
```diff
@@ -42,43 +47,60 @@ on:
...
+        then
           echo "Substitute _USER_OBJECT_ID_ placeholder value with real guid from secret"
           templateParams=${templateParams//=_USER_OBJECT_ID_/=${{ secrets.USER_OBJECT_ID }}}
+        else
+          echo "USER_OBJECT_ID not being leveraged in workflow"
         fi
-        az deployment group create -g ${{ inputs.rg }} --template-uri https://github.com/Azure/AKS-Construction/releases/download/${{ inputs.templateVersion }}/main.json --parameters ${{ inputs.templateParamFile}} automatedDeployment=true $templateParams
 
+        DEP=$(az deployment group create -g ${{ inputs.rg }} --template-uri https://github.com/Azure/AKS-Construction/releases/download/${{ inputs.templateVersion }}/main.json --parameters ${{ inputs.templateParamFile}} automatedDeployment=true $templateParams)
+        echo $DEP
```
Comment (khowling) [inline .github/workflows/AKSC_Deploy.yml:88]: lots of echos here :) and you had to get jq in :).  I'd prefer to use --query if you just want one value, you dont need any followup processing then

Comment (khowling) [reply .github/workflows/AKSC_Deploy.yml:88]: actually, we can also get acrName too, that would be nice

Comment (khowling) [reply .github/workflows/AKSC_Deploy.yml:88]: have you got a example of $DEP, and I'll write the command

**Review by khowling — CHANGES_REQUESTED** [review]

File: .github/workflows/AKSC_Deploy.yml [hunk 2]
```diff
@@ -42,43 +47,60 @@ on:
...
 
       # Deploy Bicep file
     - name: Deploy Bicep
+      id: deployAks
       env:
-        templateParams: ${{ inputs.templateParams}}
+        templateParams: "${{ inputs.templateParams}}"
       run: |
-        if [ "${{ secrets.USER_OBJECT_ID }}" ]; then
+        if [ -n "${{ secrets.USER_OBJECT_ID }}" ]
```
[the remark this answers is not in the document]
Comment (Gordonby) [reply .github/workflows/AKSC_Deploy.yml:79]: It did, with a spurious error.  Happy to revert this, found it to be a bit more explicit.

File: .github/workflows/AKSC_Deploy.yml (hunk 1 above)
Comment (Gordonby) [reply .github/workflows/AKSC_Deploy.yml:88]: I just like handling json as json not a string, besides JQ is on the runners 🎉

I don't have an example off hand.... but you can see the path here `.properties.outputs.aksClusterName.value`

File: .github/workflows/AKSC_Deploy.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (Gordonby) [reply .github/workflows/AKSC_Deploy.yml:79]: let me re-test.

**Pull request merged**